PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -86,7 +86,7 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     }
 
     if( idMD5 != calculated_md5 ) {
-      edm::LogError("IdConfigurationNotValidated")
+      edm::LogInfo("IdConfigurationNotValidated")
```

Comment:
Still trying to understand it. I am somewhat annoyed if I have to keep updating it over time for no reason...
Also, other bigger things in front of it. It's OK to turn it off for now.
I'll hopefully be able to turn it back on later.